assets/r/model_build_conditional_forest.R

```diff
@@ -82,3 +83,61 @@ ggplot(importance_df, aes(x = reorder(Variable, Importance), y = Importance)) +
   theme_minimal()
 
 dev.off()
+
+# Prepare probabilities for predictions.
+
+predicted <- kernlab::predict(model_conditionalForest, 
+                              newdata                  = tuds,
+                              type                     = "prob")[,2]
+  
+actual <- as.character(tuds[[target]])
+  
+# Create numeric risks vector.
+
+risks <- as.character(ds[[risk]])
```

Comment:
This code is repeated over and over? Use the already computed values from the model_template.R